File: pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float cash;
    int a, b, c, d, e, f, g, h;
    do
    {
        cash = get_float("Change owed: ");
    }
    while (cash <= 0);

    int cent = cash * 100;

    if (true)
    {
        a = cent / 25;
        b = cent % 25;

        c = b / 10;
        d = b % 10;

        e = d / 5;
        f = d % 5;

        g = f / 1;
        h = f % 1;

    }
    int pennies = a + c + e + g;

    printf("%i\n", pennies);






}

File: pset6/cash/cash.py
from cs50 import get_float
import sys

while True:
    cash = get_float("Change owed: ")
    #print(cash)
    if cash > 0:
        rounded = float(cash) * 100
        #print(rounded)

        a = int(rounded) / 25
        b = int(rounded) % 25

        c = int(b) / 10
        d = int(b) % 10

        e = int(d) / 5
        f = int(d) % 5

        g = int(f) / 1
        h = int(f) % 1

        pennies = int(a) + int(c) + int(e) + int(g)

        print(int(pennies))
        sys.exit(0)





    else:
        cash = get_float("Change owed: ")


